.semaphore/semaphore.yml

```diff
@@ -5,7 +5,7 @@ agent:
     type: a1-standard-4
     os_image: macos-xcode13
 execution_time_limit:
-  minutes: 90
+  minutes: 100
```

Comment:
If we are so close to the limit, may be worth to do 120 here and 115 below. 

Eventually we'll optimize / parallelize this.

Reply:
Done ✅